RELEASE.md

```diff
@@ -7,15 +7,24 @@
 ## Breaking changes to the API
 
 ## Migration guide from Kedro 0.18.* to 0.19.*
+# Upcoming Release 0.18.14
 
-# Upcoming Release 0.18.13
+## Major features and improvements
+## Bug fixes and other changes
+## Documentation changes
+## Breaking changes to the API
+## Upcoming deprecations for Kedro 0.19.0
+
+# Release 0.18.13
 
 ## Major features and improvements
-* Allowed registering of custom resolvers to `OmegaConfigLoader` through `CONFIG_LOADER_ARGS`.
 * Added support for Python 3.11. This includes tackling challenges like dependency pinning and test adjustments to ensure a smooth experience. Detailed migration tips are provided below for further context.
-* Added `kedro catalog resolve` CLI command that resolves dataset factories in the catalog with any explicit entries in the project pipeline.
+* Allowed registering of custom resolvers to `OmegaConfigLoader` through `CONFIG_LOADER_ARGS`.
 * Added support for global variables to `OmegaConfigLoader`.
-* Change Kedro starters to use `OmegaConfigLoader`.
+* Added `kedro catalog resolve` CLI command that resolves dataset factories in the catalog with any explicit entries in the project pipeline.
+* Change Kedro starters and new Kedro projects to use `OmegaConfigLoader`.
+* Adopted a flat project structure for modular pipeline and `kedro pipeline create` is updated accordingly.
```

Comment:
@noklam This is already under the Bug fixes and other changes section

Reply:
Although it's not a huge change, I would consider this as a big change since two CLI and starters has been changed (all user facing feature). 

In addition we should mention the starter structure change too?

Reply:
Let me do a final update of the release notes!